source/contributing/implementation/debugging/index.rst

```diff
@@ -0,0 +1,52 @@
+================
+Debugging Sponge
+================
+
+.. tip::
+    See :doc:`../../../plugin/debugging` for debugging a plugin and :ref:`usingDebugger` tips.
```

Comment:
Maybe use absolute paths here instead of relatives ones.

Reply:
Changed